src/content/reference/rules/index.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Reminder to update the /reference landing page, which is an index of all the reference pages and where we navigate to when you click "reference" in the navbar: https://react.dev/reference

Reply:
Ooh thanks! Will do